i18n/pt-BR.json

```diff
@@ -3688,6 +3688,10 @@
     "translation": "Exportação da conformidade para tarefa '{{.JobName}}' iniciada no '{{.FilePath}}'"
   },
   {
+    "id": "ent.data_retention.generic.license.error",
```

Comment:
Did not find this string in the translation server

Reply:
@cpanato I think the PR that added just got merged. https://github.com/mattermost/mattermost-server/pull/7539

Should be in the Translation server after the next sync, which is in a couple of hours